modules/recommended-configurable-storage-technology.adoc

```diff
@@ -91,10 +91,13 @@ recommended for {product-title} Registry cluster deployment with production work
 
 In a scaled/HA {product-title} registry cluster deployment:
 
-* The preferred storage technology is object storage. The storage technology must support RWX access mode and must ensure read-after-write consistency.
-* File storage and block storage are not recommended for a scaled/HA {product-title} registry cluster deployment with production workloads.
-* All NAS storage are not recommended for {product-title} Registry cluster
-deployment with production workloads.
+* The preferred storage technology is object storage. The storage technology must
+support RWX access mode and must ensure read-after-write consistency.
+* File storage and block storage are not recommended for a scaled/HA
+{product-title} registry cluster deployment with production workloads.
+* NAS storage is not recommended for production workloads on {product-title}
+registry clusters. The storage technology must ensure read-after-write
+consistency.
```

Comment:
If we're saying this we should require S3 compliant 'object' storage.

Reply:
NAS storage is file storage.
If you say "File storage is not recommended" , so the next sentence (NAS storage is not recommended...) is redundant.
If you want to insist especially on NAS storage, i would turn the first sentence:
"File storage (including NAS storage) and block storage are not recommended..."